main.py

```diff
@@ -56,17 +56,12 @@ def main(args: vars):
     )
 
     load_extensions(bot)
-
-    if args["clone"]:
-        bot.run(os.getenv("TOKEN_BOT_CLONE"))
-    else:
-        bot.run(os.getenv("TOKEN_BOT"))
+    bot.run(os.getenv("TOKEN_BOT"))
 
 
 if __name__ == "__main__":
     ap = argparse.ArgumentParser()
     ap.add_argument(
-        "-c",
```

Comment:
This is also correct, but incomplete. Could we remove the `--clone` argument as well, as the two are the same?

Reply:
Sure. I will update it in a bit

Reply:
Actually, please remove the whole `ap.add_argument(...) block. Not just the `--clone` line, but the entire code block regarding to the argument, starting from Line 64 to 69

Reply:
Got it

Reply:
Hey @umutucak & @PedroS235, i've updated the changes. Can you have look now.
Plus can I add a docstring to `main` function?

Reply:
You can add the docstring. Good idea.